test/conftest.py

```diff
@@ -9,6 +10,53 @@
     not mitmproxy.net.tcp.HAS_ALPN,
     reason='requires OpenSSL with ALPN support')
 
+skip_windows = pytest.mark.skipif(
+    os.name == "nt",
+    reason='Skipping due to Windows'
+)
+
+skip_not_windows = pytest.mark.skipif(
+    os.name != "nt",
+    reason='Skipping due to not Windows'
+)
+
+skip_appveyor = pytest.mark.skipif(
+    "APPVEYOR" in os.environ,
+    reason='Skipping due to Appveyor'
+)
+
+
+original_pytest_raises = pytest.raises
+
+
+def raises(exc, *args, **kwargs):
+    functools.wraps(original_pytest_raises)
```

Comment:
It's a decorator: https://docs.python.org/3/library/functools.html#functools.wraps

We can do this if you want:
```python
@functools.wraps(original_pytest_raises)
def raises(exc, *args, **kwargs):
    # ....
```